<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdtName"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 0)</string>
     </property>
     <property name="text">
      <string>职业</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 0)</string>
     </property>
     <property name="text">
      <string>姓名</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 0)</string>
     </property>
     <property name="text">
      <string>生日</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBoxGender">
     <item>
      <property name="text">
       <string>男</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>女</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>保密</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDateEdit" name="dateEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 0)</string>
     </property>
     <property name="text">
      <string>性别</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBoxJob">
     <item>
      <property name="text">
       <string>魔法师</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>矮人</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>精英怪</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>哥布林</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>狂战士</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>弓箭手</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1" alignment="Qt::AlignmentFlag::AlignTop">
    <widget class="QPushButton" name="pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::RightToLeft</enum>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
